install-vdvs.ps1

```diff
@@ -20,7 +20,8 @@
 
 # Command line parameters
 param (
-    [Parameter(Mandatory=$true)][string]$uri
+    [string]$uri, # TODO: add a default value once it's available
```

Comment:
Two TODOs referring to the default value of the uri. If it is worth, lets track it through an issue and use the issue number in the comment.